Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1725,14 +1726,19 @@ def convert_os_to_standard(endpoint_os):
     return os_type
 
 
+def get_endpoint_properties(single_endpoint):
+    status = 'Online' if single_endpoint.get('endpoint_status').lower() == 'connected' else 'Offline'
+    is_isolated = 'No' if 'unisolated' in single_endpoint.get('is_isolated', '').lower() else 'Yes'
+    hostname = single_endpoint['host_name'] if single_endpoint.get('host_name', '') else single_endpoint.get(
```

Comment:
no need to alter default
```suggestion
    hostname = single_endpoint['host_name'] if single_endpoint.get('host_name') else single_endpoint.get(
```